apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -474,6 +475,16 @@ rewards_contract_pubkey() ->
           end,
     aeu_ets_cache:get(?ETS_CACHE_TABLE, rewards_contract_pubkey, Fun).
 
+staking_contract_pubkey() ->
+    Fun = fun() ->
+              EncContractId = get_consensus_config_key([<<"staking_contract">>]),
+              case aeser_api_encoder:safe_decode(contract_pubkey, EncContractId) of
+                  {ok, Pubkey} -> Pubkey;
+                  _ -> erlang:error({contract_owner_not_valid_contract, EncContractId})
+              end
+          end,
+    aeu_ets_cache:get(?ETS_CACHE_TABLE, rewards_contract_pubkey, Fun).
```

Comment:
Why here the rewards contract pubkey as default???

Reply:
CopyPaste error, will fix. ran since rewards and staking are the same in the current setup.